### PR TITLE
Use plugin occluder instead of model occluder

### PR DESCRIPTION
**Pull request by RomanFrom710**
Added occluder model to the Unity plugin (I got .obj file from [this](https://www.creators3d.com/online-viewer) converter).

### CODE REVIEW

**Comment by brentchow** [comment]
@RomanFrom710 @calvertcreates does `.obj` support blend shapes?

**Comment by calvertcreates** [comment]
@brentchow Short answer is no.

**Comment by brentchow** [comment]
> @brentchow Short answer is no.

@RomanFrom710 we shouldn't use`.obj` if it doesn't include the blend shapes, but rather, use a `.gltf` or other format that allows us to maintain the blend shapes in the `Occluder`.

**Comment by RomanFrom710** [comment]
Currently we don't support blendshapes in Unity. I added it as `obj` since Unity has native support of it, so we bypassing using of GLTF plugin. We can switch to `gltf` occluder any time since it's internal and won't break anything.

**Comment by RomanFrom710** [comment]
Replace occluder with `.fbx`: it both supports blendshapes and is supported by Unity

File: Assets/Svrf/Scripts/Utilities/SvrfModelUtility.cs [hunk 1]
```diff
@@ -31,28 +31,19 @@ internal static async Task AddSvrfModel(GameObject gameObject, MediaModel model,
...
-                var meshRenderer = occluder.transform.Find("Primitive").GetComponent<SkinnedMeshRenderer>();
-                // If we need to handle occlusion, apply our custom shader to handle it.
-                meshRenderer.sharedMaterials[0].shader = Shader.Find("Svrf/Occluder");
-            }
-            else
-            {
-                // If we don't need to handle occlusion, hide the occluder.
-                occluder.gameObject.SetActive(false);
+                var occluder = Object.Instantiate(Resources.Load<GameObject>("Occluder"));
+                occluder.transform.parent = gltfRoot;
```
Comment (ox) [inline Assets/Svrf/Scripts/Utilities/SvrfModelUtility.cs]: Does this not need a shader?

Comment (RomanFrom710) [reply Assets/Svrf/Scripts/Utilities/SvrfModelUtility.cs]: The shader is already applied to the material that is used by `Occluder` game object.

**Pull request merged**